docs/Using Fleet/segment-hosts.md

```diff
@@ -6,128 +6,52 @@
 ℹ️  In Fleet 4.0, Teams were introduced.
 ```
 
-- [View teams](#view-teams)
-- [Create a team](#create-a-team)
-- [Automatically adding hosts to a team](#automatically-adding-hosts-to-a-team)
+- [Overview](#overview)
+- [Naming conventions](#naming-conventions)
 - [Transfer hosts to a team](#transfer-hosts-to-a-team)
-- [Add users to a team](#add-users-to-a-team)
-- [Remove a member from a team](#remove-a-member-from-a-team)
-- [Remove a team](#remove-a-team)
+
+## Overview
 
 In Fleet, you can group hosts together in a team.
 
 With hosts segmented into exclusive teams, you can apply specific queries, policies, and agent options to each team.
 
-For example, you might create a team for each type of system in your organization. You can name the teams `Workstations`, `Workstations - sandbox`, `Servers`, and `Servers - sandbox`.
-
-> A popular pattern is to end a team’s name with “- sandbox”, then you can use this to test new queries and configuration with staging hosts or volunteers acting as canaries.
-
 Then you can:
 
 - Enroll hosts to one team using team specific enroll secrets
 
 - Apply unique agent options to each team
 
-- Schedule queries that target one or more teams
-
-- Run live queries against one or more teams
-
-- Grant users access to one or more
+- Schedule queries that target one or more team
 
-## View teams
+- Run live queries against one or more team
 
-To view teams:
+- Grant users access to one or more team
 
-In the top navigation select "Settings" and then "Teams."
+You can manage teams in the Fleet UI by selecting **Settings** > **Teams** in the top navigation. From there, you can add or remove teams, manage user access to teams, transfer hosts, or modify team settings.
 
-## Create a team
+## Naming conventions
 
-To create a team:
+One recommended approach is to create a team for each type of system in your organization. For example, you may have teams named: `Workstations`, `Workstations - sandbox`, `Servers`, and `Servers - sandbox`.
```

Comment:
```suggestion
The best practice is to create these teams: `Workstations`, `Workstations (canary)`, `Servers`, and `Servers (canary)`.
```

I think we can make this language opinionated and call it the best practice.

I also updated the team names so they're up to date w/ current best practice.

What do you think?